Integrations/integration-Tanium.yml

```diff
@@ -556,40 +816,57 @@ script:
     outputs:
     - contextPath: Tanium.Packages.verify_expire_seconds
       description: Timeout for verify action expiry in seconds
+      type: string
     - contextPath: Tanium.Packages.display_name
       description: Display name of package
+      type: string
     - contextPath: Tanium.Packages.name
       description: Name of created package
+      type: string
     - contextPath: Tanium.Packages.command
       description: Command to execute
+      type: string
     - contextPath: Tanium.Packages.creation_time
       description: Pacakge creation time
+      type: date
     - contextPath: Tanium.Packages.command_timeout
       description: Timeout for command execution in seconds
+      type: number
     - contextPath: Tanium.Packages.modification_time
       description: Package modification time
+      type: date
     - contextPath: Tanium.Packages.last_update
       description: Time when package was last updated
+      type: date
     - contextPath: Tanium.Packages.deleted_flag
       description: Is the package deleted
+      type: boolean
     - contextPath: Tanium.Packages.id
       description: Tanium unique package id
+      type: string
     - contextPath: Tanium.Packages.expire_seconds
       description: Timeout for action expiry in seconds
+      type: number
     description: Create a package object
   - name: tn-approve-pending-action
     arguments:
-    - name: action_id
-      required: true
+    - name: saved_action_id
+      description: Saved action ID to be approved
+    - name: id
       default: true
-      description: Id of saved action to approve
+      description: Action ID to be approved
```

Comment:
this is not always to be approved..

Reply:
@Bars92  it's an argument of tn-approve-pending-action, it has to be "to be approved"